library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java

```diff
@@ -42,8 +61,12 @@
     public static final String CATEGORY_CONTENT_EVENT = "Content Event";
     private Tracker mTracker = null;
 
+    private WeakReference<Activity> activity;
```

Comment:
Android naming convention for properties is:
`private WeakReference<Activity> mActivity`